cdap-watchdog/src/main/java/co/cask/cdap/logging/appender/LogAppender.java

```diff
@@ -20,11 +20,37 @@
 import ch.qos.logback.core.AppenderBase;
 import co.cask.cdap.common.logging.LoggingContext;
 import co.cask.cdap.common.logging.LoggingContextAccessor;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
 /**
  * CDAP log appender interface.
  */
 public abstract class LogAppender extends AppenderBase<ILoggingEvent> {
+  private static final Logger LOG = LoggerFactory.getLogger(LogAppender.class);
+  private static final String USER_LOG_TAG = ".userLog";
```

Comment:
Do we want to separate out program log and plugin log as well? Or all of them are just "user" log? Also, it's not really "user" log, right?